packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -383,19 +415,18 @@ class Radio extends React.Component<Props> {
                 ),
             };
         }
-        /* jshint -W018 */
-        const correct = _.all(userInput.choicesSelected, (selected, i) => {
+
+        const correct = userInput.choicesSelected.every((selected, i) => {
             let isCorrect;
             if (userInput.noneOfTheAboveIndex === i) {
-                isCorrect = _.all(rubric.choices, (choice, j) => {
```

Comment:
**praise:** thanks for removing uses of `lodash`.  Having 100% Flow coverage provides a high level of confidence in changes like this.